observability/otel/metrics.ts

```diff
@@ -1,3 +1,4 @@
+import type { Meter } from "npm:@opentelemetry/api@1.9.0";
```

Comment:
you shouldn't import from npm directly, import from `deps.ts`

Reply:
I tried to use MeterProvider from deps.ts, but got this error:

`Type 'Meter' is missing the following properties from type 'MeterProvider': _sharedState, _shutdown, getMeter, addMetricReader, and 2 more.`

Reply:
Because there are more than one definition of meter, you need to import it correctly from opentelemetry/api.